### PR TITLE
fix(swap): blink when re-fetch aggregator API

### PR DESCRIPTION
**Pull request by nguyenhuudungz**
Deadline doesn't need to be based on `currentBlockTimestamp`. Instead, `Date.now()` is a better option.

### CODE REVIEW

File: src/utils/aggregator.ts [hunk 1]
```diff
@@ -265,16 +271,18 @@ export class Aggregator {
...
     currencyAmountIn: CurrencyAmount<Currency>,
     currencyOut: Currency,
     slippageTolerance: number,
-    deadline: BigNumber | undefined,
+    deadline: number | undefined,
     to: string,
     feeConfig: FeeConfig | undefined,
-    signal?: AbortSignal,
+    signal: AbortSignal,
+    aggregatorWaitingTime: number,
```
Comment (viet-nv) [inline src/utils/aggregator.ts:285]: I think we should change name of this variable. maybe `minimumLoadingTime`

Comment (nguyenhuudungz) [reply src/utils/aggregator.ts]: Done.

**Review by viet-nv — APPROVED** [review]

File: src/utils/fetchWaiting.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default async function fetchWaiting(input: RequestInfo, init?: RequestInit, minimumLoadingTime = 0) {
+  const startTime = Date.now()
+  const response = await fetch(input, init)
+  const endTime = Date.now()
+  const timeoutTime = minimumLoadingTime - (endTime - startTime)
+  await new Promise(resolve => setTimeout(resolve, timeoutTime))
+  return response
```
Comment (namgold) [inline src/utils/fetchWaiting.ts:7]: another way to achieve is create a `setTimeout` with `minimumLoadingTime`, then `Promise.all` it with `fetch`. This solution is much straighforward IMO, dont need to calc time left.

Comment (viet-nv) [reply src/utils/fetchWaiting.ts:7]: awesome

Comment (hungdoansy) [reply src/utils/fetchWaiting.ts:7]: Agree

**Pull request merged**